frontend/components/Cell.js

```diff
@@ -167,6 +168,7 @@ export const Cell = ({
                 selected: selected,
                 code_differs: class_code_differs,
                 code_folded: class_code_folded,
+                metadata: metadata,
```

Comment:
This is just for the css metadata class, if the cell has metadata, but it's always true, so I'm removing for now! If you had a use case in mind, shout!

Reply:
In principle, cell formatting could in the future be dependent on cell metadata, e.g. allowing different cell widths. But if something like this is introduced, passing metadata to the class could be re-introduced.

Reply:
Yes, and most likely it will be something inside the metadata, not the whole object (which now is always true-ish, so cells get a "metadata" class) let's add specific data from metadata like "cell-width" for example etc